commitment/encoding.go

```diff
@@ -161,6 +161,10 @@ func TapscriptPreimageDecoder(r io.Reader, val any, buf *[8]byte,
 		return tlv.ErrRecordTooLarge
 	}
 
+	if l < 1 {
```

Comment:
This mean to check `== 0`?

Reply:
Yes. I guess that's more clear, changed it.